src/vsl/parser/grammar/class.ne

```diff
@@ -43,6 +43,6 @@ ClassItem[s]
    -> FunctionStatement[$s {% id %}] {% id %}
 
 InterfaceItems[s]
-   -> CodeBlock[(InterfaceItem | ClassItem[$s {% id %}]) {% mid %}] {% id %}
```

Comment:
This should still be there, as interfaces can have default implementations of methods:

```swift
public interface Comparable<T>: Equatable<T>, Constrastable<T> {
     public static func >=(lhs: T, rhs: T) { default implementation }
}
```

Though that said. Interfaces can't have fields so this would probably have to have a specific exception to allow for functions.